<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QDialog" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>960</width>
     <height>660</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(30,31,46,0.70);
border-radius: 80px;
opacity:</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
color: #FFFFFF;
background: rgba(0,0,0,0.0)
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:48px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:48pt; font-weight:400;&quot;&gt;Create An Account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Username">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>329</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
display: flex;
align-items: center;
text-align::placeholder: center;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Age">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>329</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
display: flex;
align-items: center;
text-align: center;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>329</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
display: flex;
align-items: center;
text-align: center;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>329</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
display: flex;
align-items: center;
text-align: center;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Passsword2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>329</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
display: flex;
align-items: center;
text-align: center;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>360</y>
     <width>329</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #D9D9D9;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 48px;
line-height: 58px;
display: flex;
align-items: center;
text-align: center;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>457</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #BB5858;
display: flex;
border-radius: 30px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 58px;
line-height: 70px;
align-items: center;
text-align: center;

color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>textEdit</zorder>
  <zorder>Username</zorder>
  <zorder>Age</zorder>
  <zorder>Password</zorder>
  <zorder>Name</zorder>
  <zorder>Passsword2</zorder>
  <zorder>Email</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
